docs/DeveloperGuide.md

```diff
@@ -283,11 +283,34 @@ Priorities: High (must have) - `* * *`, Medium (nice to have) - `* *`, Low (unli
 
 ### Use cases
 
-(For all use cases below, the **System** is the `AddressBook` and the **Actor** is the `user`, unless specified otherwise)
```

Comment:
I think these changes would probably cause a merge conflict? Because you had also added the use cases in a different PR

Reply:
Okay, got it. Will change.